tari_wallet_lib/src/lib.rs

```diff
@@ -326,4 +325,13 @@ pub fn view_vault_balance(
             .collect();
     
     Ok(serde_wasm_bindgen::to_value(&result)?)        
+}
+
+// serde-wasm has some limitations when the structs use the serde's "flatten" macro
```

Comment:
Interesting, flatten creates quite a few issues it seems. It was done in the transaction struct to avoid having to change downstream clients but I think eventually we'll need to do that refactor when we secure multiple signers properly.